### DISCUSSION TITLE
怎么直接播放链接啊

### DISCUSSION THREAD

**Discussion by yzjdev** [hooke007/mpv_PlayKit#358] [mpv 通用]
百度过了， 实在找不到怎么直接播放链接啊， 官方文档英文 直接看不懂， 大佬的懒人mpv， 可是我找了半天也还是没找到关于播放链接的， 我好废啊

**Comment by hooke007** [comment]
Also see
https://github.com/hooke007/MPV_lazy/discussions/295#discussioncomment-5669209
https://github.com/hooke007/MPV_lazy/discussions/310#discussioncomment-5977965
